network-api/networkapi/wagtailpages/factory/libraries/research_hub/__init__.py

```diff
@@ -18,6 +18,20 @@
 from networkapi.wagtailpages.factory.libraries.research_hub import (
     taxonomies as taxonomies_factory,
 )
+from networkapi.wagtailpages.pagemodels.profiles import Profile
+
+
+def create_detail_page_for_visual_regression_tests(seed, research_library_page):
+    percy_author_profile = Profile.objects.get(name="Percy Profile")
```

Comment:
`seed` didn't get used here. Did you intend to use it to reseed?

Reply:
Thanks for the catch Mavis! I have added `reseed(seed)` to the file on the latest commit of the branch.